packages/vue-cli-plugin-typescript/generator/index.js

```diff
@@ -12,11 +12,15 @@ module.exports = (api, options, rootOptions) => {
       '@coveo/headless': '*',
       '@coveo/search-token-server': '*',
       buefy: '^0.9.4',
+      bulma: '^0.9.3',
       concurrently: '^5.3.0',
       dotenv: '^9.0.2',
       'get-port': '^5.1.1',
       'fs-extra': '^10.0.0',
     },
+    devDependencies: {
+      tslib: '^2.3.0',
```

Comment:
Same thing as above.
tslib should be added by [@vue/cli](https://cli.vuejs.org/), when using a typescript preset.